sansio_lsp_client/structs.py

```diff
@@ -83,10 +90,20 @@ def calculate_length(self, text: str) -> int:
 class TextDocumentContentChangeEvent(BaseModel):
     text: str
     range: t.Optional[Range]
-    rangeLength: t.Optional[int]
+    rangeLength: t.Optional[int]  # deprecated, use .range
+
+    def dict(self) -> t.Dict[str, t.Any]:
+        d = super(TextDocumentContentChangeEvent, self).dict()
```

Comment:
Why the full spelling of `super(...)`?

Reply:
I just wasn't very familiar with `super` at that point :)

Reply:
Changed in #41